src/track/serato/markers2.cpp

```diff
@@ -2,6 +2,23 @@
 
 #include <QtEndian>
 
+namespace {
+QString zeroTerminatedUtf8StringtoQString(QDataStream* stream) {
+    DEBUG_ASSERT(stream);
+
+    QByteArray data;
+    quint8 byte = '\xFF';
+    while (byte != '\x00') {
+        *stream >> byte;
+        data.append(byte);
```

Comment:
Do we have test that the byte order is correct?

Reply:
Yes: https://github.com/mixxxdj/mixxx/blob/26fc46fef090d964cf1ab37065f759b75fccf393/src/test/seratomarkers2test.cpp#L114-L125
We also have multiple tests that read cues with labels and ensure that these cues have the same label after `dump()`.